src/app/AppFooter/AppFooter.js

```diff
@@ -1,23 +1,22 @@
 import styled from 'styled-components/macro'
 import Navigation from '../Navigation'
 
-export default function AppFooter({className}) {
-
+export default function AppFooter({ className }) {
   return (
-    <FooterStyled className={className}>
-      <Navigation/>
-    </FooterStyled>
+    <Footer className={className}>
+      <Navigation />
+    </Footer>
   )
 }
 
-const FooterStyled = styled.footer`
+const Footer = styled.footer`
+  height: 50px;
+  max-width: 1240px;
   margin: 0 auto;
   display: flex;
-  justify-content: space-evenly;
   align-items: center;
-  height: 50px;
-  max-width: 1240px;
-  box-shadow: 0 0 10px var(--primary-dark);
-  border-radius: 25px 25px 0 0 ;
-  background: var(--gradient-dark);
-`
\ No newline at end of file
+  justify-content: space-evenly;
+  box-shadow: 0 0 5px var(--secondary-medium);
```

Comment:
You could think about a consistent shadow variable. So you don't need to repeat the values every time.